docs/import_hooks.md

```diff
@@ -219,18 +255,20 @@ WORDPRESS_IMPORT_HOOKS_TAGS_TO_CACHE = {
     # the key is the XML tag name of the item tag that contains the related content
     "wp:author": {
         # the XML tag which has the value to lookup in the tags cache
-        "DATA_TAG": "dc:creator",
+        "DATA_TAG": "dc_creator",
         # the dotted path to the function to call in your own Wagtail site
         "FUNCTION": "pages.import_hooks.author_processor",
     }
 }
 ```
 
-The key of the dict is the name for the tag/s to cache in the XML file which contains the related content.
+The key of `wp:author` is the tag name in the XML file.
 
-`DATA_TAG` is the tag `<dc:creator></dc:creator>` value. *This is passed to your function.*
+The `DATA_TAG` value is the key to use to look up the value for the author in your own function that processes the author relation. This value is passed to your function.
 
-`FUNCTION` is the dotted path to the function that will be called.
+**Important:** If the DATA_TAG value in your XML has a leading `_` character do not include the underscore in the value above. If the value contains a `:` then replace it with a `_` in the value above. This is because in the function that processes the header image, the value is used to construct a lookup by key query parameter on a JSONField available on the page model.
```

Comment:
**Comment**: It seems reasonable to flag this here. Can we also tell implementers that there's a function `wagtail_wordpress_import.functions.snakecase_key` if they want to use it?

Reply:
**Question** Is there a specific place you see that might be useful to them? I could see it been useful in the function they provide. Do you mean add the note there?

or

We could do away with the note entirely here and elsewhere and leave the value of `"DATA_TAG": "dc_creator"` as `"DATA_TAG": "dc:creator"` and tell them to use the function to change the param `data_tag` in their own function.

Reply:
I don't know. I'm losing track of where code is used, and what the audience is for the docs. Use your judgement here. Sorry.

Reply:
On the basis that we will need to document this or the usage of a function we provide I think this is a good place. 

I'd prefer to the leave this in place and not encourage them to use a function that we provide. Making the change to the value when setting the config is easier to explain over explaining about using a function, I think.